tests/run/cpp_unordered_map_helper.h

```diff
@@ -0,0 +1,12 @@
+#include <vector>
+
+struct IntVectorHash {
+  size_t operator()(const std::vector<int>& v) const {
+    std::hash<int> hasher;
```

Comment:
Do you need to include something to ensure `std::hash` is available? I know it's probably included with `unordered_map` but I don't know that you can/should 100% rely on the order Cython includes stuff.